app/static/src/scss/includes/_variables.scss

```diff
@@ -0,0 +1,20 @@
+// breakpoints
+$breakpoint-small: 640px;
+$breakpoint-medium: 810px;
+$breakpoint-large: 1200px;
+
+// colour__component__type
+$colour-link-normal: #0b0c0c;
```

Comment:
checking the GDS colour standards I believe normal links should have a colour of #1d70b8, then visited links #4c2c92 https://design-system.service.gov.uk/styles/colour